static_files/css/style.css

```diff
@@ -66,24 +66,41 @@ li {
 
 .VisibleMenu {
 
-    color: #CCCCCC;
+    color: #BCBF12;
     font-weight: bold;
+    display: flex;
+    align-items: center;
 }
 
+
 .VisibleMenu a {
+
     color: #BCBF12;
 }
 
+.sandwich-button {
+    padding-top: 6px;
+}
```

Comment:
I assume this spacing is to align the icon with the rest of the list items. I believe this misalignment is due to the type of font used. In that case, changing the font to nunito and using a flex display would be a workaround. The purpose of this point is more for learning purposes :)

Reply:
Fixed according to the suggestions we've discussed on Slack